systemtest/src/main/java/io/strimzi/systemtest/templates/crd/KafkaTopicTemplates.java

```diff
@@ -4,57 +4,71 @@
  */
 package io.strimzi.systemtest.templates.crd;
 
-import io.strimzi.api.kafka.model.topic.KafkaTopic;
 import io.strimzi.api.kafka.model.topic.KafkaTopicBuilder;
 import io.strimzi.operator.common.model.Labels;
-import io.strimzi.systemtest.TestConstants;
 import io.strimzi.systemtest.storage.TestStorage;
-import io.strimzi.test.TestUtils;
 
 public class KafkaTopicTemplates {
 
     private KafkaTopicTemplates() {}
 
     public static KafkaTopicBuilder topic(TestStorage testStorage) {
-        return defaultTopic(testStorage.getClusterName(), testStorage.getTopicName(), 1, 1, 1, testStorage.getNamespaceName());
+        return defaultTopic(testStorage.getNamespaceName(), testStorage.getTopicName(), testStorage.getClusterName(), 1, 1, 1);
     }
 
     public static KafkaTopicBuilder continuousTopic(TestStorage testStorage) {
-        return defaultTopic(testStorage.getClusterName(), testStorage.getContinuousTopicName(), 1, 1, 1, testStorage.getNamespaceName());
+        return defaultTopic(testStorage.getNamespaceName(), testStorage.getClusterName(), testStorage.getContinuousTopicName(), 1, 1, 1);
     }
 
-    public static KafkaTopicBuilder topic(String clusterName, String topicName, String topicNamespace) {
-        return defaultTopic(clusterName, topicName, 1, 1, 1, topicNamespace);
+    public static KafkaTopicBuilder topic(String namespaceName, String topicName, String clusterName) {
```

Comment:
also I know it can be a bit tedious but it seems there may be some leftovers from refactoring all arguments e.g. https://github.com/strimzi/strimzi-kafka-operator/pull/10418/files#diff-e634cc1363eb785e6f6f291dab64e377e33a268df7d700d583049ca5f12c5241L612 seems to be in wrong order.

Reply:
Let me check, thanks.

Reply:
resolved, good catch :)